CMSgov/mint-app
# Easi 2186/it tools warning

# EASI-2186

## Changes and Description

- Created custom warning component for relevant IT tools questions
- Updated cypress test to utilize new warning reroute functionality
- Mobile responsiveness

![Screen Shot 2022-08-15 at 2 08 30 PM](https://user-images.githubusercontent.com/95709965/184691167-b9d6e879-8a17-462e-ae44-235e714c58a4.png)

## How to test this change

- Create/open a model plan
- Ensure that IT tools status is not `READY`
- Navigate to any IT tools questions and click 'Go to question'
- Ensure the warning is displayed next to relevant question
- Ensure the link in the warning navigates back to the corresponding IT tools question

NOTE***
There a a few questions that span multiple IT tools pages, so the warning redirect may* bring you to an undesired page.  I have consulted with Natasha, and this side effect is fine.  For example 'What data do you need to monitor the model? Select all that apply.' Appears on both page 5 and 6 of IT tools.  The warning component will never have the context of the desired IT tools page (The person viewing the warning will not always be coming from IT tools, so this effect is inevitable).

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
